src/services/doppler-user-api-client.ts

```diff
@@ -101,9 +102,24 @@ export class HttpDopplerUserApiClient implements DopplerUserApiClient {
     }
   }
 
-  public async createOrUpdateContactInformation(
-    values: any,
-  ): Promise<EmptyResultWithoutExpectedErrors> {
-    return { success: true };
+  public async updateContactInformation(values: any): Promise<EmptyResultWithoutExpectedErrors> {
+    try {
+      const { email, jwtToken } = this.getDopplerUserApiConnectionData();
+
+      const response = await this.axios.request({
+        method: 'POST',
```

Comment:
Why not `PUT`?

Reply:
when we discussed with Andres about Users Api, we decided that use POST for this endpoint.